Common/src/CConfig.cpp

```diff
@@ -6342,7 +6342,6 @@ void CConfig::SetOutput(unsigned short val_software, unsigned short val_izone) {
       if (TimeMarching == DT_STEPPING_2ND) cout << "Unsteady simulation, dual time stepping strategy (second order in time)."<< endl;
       if (Unst_CFL != 0.0) cout << "Time step computed by the code. Unsteady CFL number: " << Unst_CFL <<"."<< endl;
       else cout << "Unsteady time step provided by the user (s): "<< Delta_UnstTime << "." << endl;
-      cout << "Total number of internal Dual Time iterations: "<< InnerIter <<"." << endl;
```

Comment:
Is this output not relevant?

Reply:
I think it is redundant as the number of inner iterations are displayed in any case. Plus, the number of inner iterations is actually nInnerIter, thus that output always display Total number of internal Dual Time iterations: 0
Is it maybe a legacy of old implementations?